core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -223,6 +239,39 @@ oppia.controller('FeedbackTab', [
               });
             };
 
+            $scope.editSuggestion = function() {
+              $scope.suggestionEditorIsShown = true;
+              $scope.isNotHandled = true;
+              $scope.saveButtonIsShown = true;
+            };
+
+            $scope.saveSuggestion = function() {
+              $scope.suggestionEditorIsShown = false;
+              $scope.saveButton = false;
+
+              url = UrlInterpolationService.interpolateUrl(
```

Comment:
This looks similar to the functionality in `CreatorDashboard.js`, do we have any service for handling HTTP request for suggestions? 

/cc @nithusha21